lib/pf/Switch/Cisco/Catalyst_2960.pm

```diff
@@ -542,6 +557,104 @@ sub returnRadiusAccessAccept {
     return [$status, %$radius_reply_ref];
 }
 
+=head2
+
+Return Radius reply in the access-challenge
+
+=cut
+
+sub returnRadiusAdvanced {
```

Comment:
If this is the same code than in the ASA module, I think it would make sense to reuse it instead of copy/pasting it